orc8r/cloud/go/obsidian/swagger/v1/swagger.yml

```diff
@@ -8747,6 +8747,57 @@ definitions:
     - macs
     - ips
     type: object
+  log:
+    properties:
+      body:
+        description: Body of log
+        example: '{[{"registrationRequest":{"cbsdSerialNumber":"some_serial_number"}}]}'
+        type: string
+        x-nullable: false
+      fcc_id:
+        description: Fcc Id of cbsd involved in log
+        example: some_fcc_id
+        type: string
+        x-nullable: false
+      from:
+        description: log origin
+        enum:
+        - SAS
```

Comment:
Can you please fix these linting issues?

Reply:
This is autogenerated code by `python ${MAGMA_ROOT}/orc8r/cloud/docker/build.py --generate` and if you changed it, insync job (which checks if generated files are exactly the same) would  fail

Reply:
Okay, thanks.